projects/demo/src/modules/components/input/examples/3/index.html

```diff
@@ -12,11 +12,14 @@
     <p>
         <tui-input formControlName="testValue1" [textMask]="textMaskOptions1">
             Document number
+            <input tuiTextfield />
         </tui-input>
     </p>
     <p>
         <tui-input formControlName="testValue2" [textMask]="textMaskOptions2">
             Secret number
+            <input tuiTextfield />
         </tui-input>
     </p>
 </form>
+<pre>{{testForm.value | json}}</pre>
```

Comment:
debug

Reply:
We can probably leave it here to show that form control values are the same as masked input value.

Reply:
please, add an html comment that this is necessary for the demonstration